appveyor.yml

```diff
@@ -7,13 +7,13 @@ environment:
 version: reactos.appveyor.{build}
 skip_branch_with_pr: true
 clone_depth: 5
-clone_folder: c:\reactos-cov
+clone_folder: C:\reactos-cov
```

Comment:
This is a workaround. Fix configure.cmd to use `IF /I` instead.

Reply:
Added 2 `configure.cmd` commits...